.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Thank you for contributing to our devenv project!
+
+It would be helpful, if you gave us as much information as you could to process your pull request better.
+Therefore you are given this description template.
+-->
+
+### 1. Why is this change necessary?
+
+### 2. What does this change do, exactly?
+
+### 3. Describe each step to reproduce the issue or behaviour.
+
+### 4. Please link to the relevant issues (if any).
+
+### 5. Checklist
+
+- [ ] I have rebased my changes to remove merge conflicts
+- [ ] I have written or adjusted the documentation according to my changes
```

Comment:
```suggestion
- [ ] I have written or adjusted the README according to my changes
```

Reply:
due to #33 we have to revert this change 🙈 
Sorry for the inconvenience

Reply:
It's okay ^^
Reverted the change. 👍